account_reconciliation_not_stupid/models/res_partner.py

```diff
@@ -17,3 +17,10 @@ def open_payment_matching_screen(self):
             "tag": "manual_reconciliation_view",
             "context": action_context,
         }
+
+    customer = fields.Boolean(
```

Comment:
No you can't do that, you need to adapt to how the code has changed, not just put the old removed code back in another module.